containers/ddev-webserver/ddev-webserver-base-files/usr/local/bin/enable_xhprof

```diff
@@ -4,6 +4,13 @@ phpdismod blackfire xdebug
 mkdir -p ${XHPROF_OUTPUT_DIR}
 phpenmod xhprof
 killall -USR2 php-fpm 2>/dev/null || true
-echo "Enabled xhprof.
-After each web request or CLI process you can see all runs, most recent first, at
+echo "Enabled xhprof with xhprof_mode=${DDEV_XHPROF_MODE}"
+case "${DDEV_XHPROF_MODE}" in
+  xhgui)
+    echo "Use 'ddev xhgui' to launch browser for xhprof results."
```

Comment:
The image needs a new push, I see the old message `to lanch browser`

Reply:
Done, thanks.